org.nanuc.Axolotl.yml

```diff
@@ -39,168 +29,71 @@ finish-args:
       env:
-        GOBIN: /app/bin
-        GO111MODULE: off
-        GOPATH: /run/build/astilectron-bundler
-    subdir: src/github.com/asticode/go-astilectron-bundler
-    build-commands:
-      - go install -v github.com/asticode/go-astilectron-bundler/...
-    sources:
-      - generated-astilectron-bundler-sources.json
-      - type: git
-        url: https://github.com/asticode/go-astilectron-bundler
-        tag: v0.7.12
-        commit: 9ca0b71fdf5533e4aa17f7587fb747d3874a7315
-        dest: src/github.com/asticode/go-astilectron-bundler
-        x-checker-data:
-          type: git
-          tag-pattern: "^v([\\d.]+)$"
-
-  - name: astilectron-bundler-astilectron
-    buildsystem: simple
-    build-options:
-      env:
-        BUNDLER_CACHE: /app/lib/astibundler/cache
-    build-commands:
-      - install -Dm 644 astilectron-0.56.0.zip ${BUNDLER_CACHE}/astilectron-0.56.0.zip
-    sources:
-      - type: file
-        url: https://github.com/asticode/astilectron/archive/v0.56.0.zip
-        sha256: 2725492843edaef0530543c041c5c7d9e43d4163f5c192fc8da11ce89fcaa0f5
-        dest-filename: astilectron-0.56.0.zip
-
-  - name: astilectron-bundler-electron-amd64
-    buildsystem: simple
-    only-arches:
-      - x86_64
-    build-options:
-      env:
-        BUNDLER_CACHE: /app/lib/astibundler/cache
-    build-commands:
-      - install -Dm 644 electron-linux-amd64-20.2.0.zip ${BUNDLER_CACHE}/electron-linux-amd64-20.2.0.zip
-    sources:
-      - type: file
-        url: https://github.com/electron/electron/releases/download/v20.2.0/electron-v20.2.0-linux-x64.zip
-        sha256: c0e3522de34819b838f4a35ddf30c6283c61be1bb8dff02089cda3f641938aad
-        dest-filename: electron-linux-amd64-20.2.0.zip
-
-  - name: astilectron-bundler-electron-arm64
-    buildsystem: simple
-    only-arches:
-      - aarch64
-    build-options:
-      env:
-        BUNDLER_CACHE: /app/lib/astibundler/cache
-    build-commands:
-      - install -Dm 644 electron-linux-arm64-20.2.0.zip ${BUNDLER_CACHE}/electron-linux-arm64-20.2.0.zip
-    sources:
-      - type: file
-        url: https://github.com/electron/electron/releases/download/v20.2.0/electron-v20.2.0-linux-arm64.zip
-        sha256: 1d2dd05f060786eae5ce687794aa071df8a5f5733b7b2efa1481de8db75d7738
-        dest-filename: electron-linux-arm64-20.2.0.zip
-
-  - name: axolotl-electron-bundle
-    buildsystem: simple
-    build-options:
-      env:
-        npm_config_offline: "true"
-        npm_config_cache: /run/build/axolotl-electron-bundle/flatpak-node/npm-cache
-        GOBIN: /app/bin
-        GO111MODULE: off
-        GOPATH: /run/build/axolotl-electron-bundle
-    subdir: src/github.com/nanu-c/axolotl
+        # Cargo config
+        CARGO_HOME: /run/build/axolotl/cargo
+        # Cargo offline mode
+        CARGO_NET_OFFLINE: "true"
     build-commands:
+      # Build axolotl-web
       - make build-dependencies-axolotl-web
       - make build-axolotl-web
```

Comment:
Seems like I need to wait for https://github.com/nanu-c/axolotl/pull/1173 to be merged to fix the `Makefile`.

Reply:
I upgraded to the changes from nanu-c/axolotl#1192 which only uses targets from the `Makefile`.